ChangeLog

```diff
@@ -1,6 +1,8 @@
 CHANGES
 =======
 
```

Comment:
You don't need to modify change log manually, it will be generated by bumping a new tag